src/plugins/embeddable/public/lib/panel/panel_header/panel_actions/customize_panel/customize_panel_action.tsx

```diff
@@ -129,6 +129,10 @@ export class CustomizePanelAction implements Action<CustomizePanelActionContext>
       {
         size: 's',
         'data-test-subj': 'customizePanel',
+        onClose: (overlayRef) => {
+          if (overlayTracker) overlayTracker.clearOverlays();
+          overlayRef.close();
+        },
```

Comment:
This fixes a small bug I discovered with the panel settings flyout where, if you closed the flyout by clicking outside of the flyout rather than via the "Close" buttons, `hasOverlays` remains `true` and so a bunch of items in the top nav bar stayed disabled.

**Before**

![Apr-13-2023 15-28-16](https://user-images.githubusercontent.com/8698078/231886930-6dde8760-cc89-4613-accb-b956d3c14a8b.gif)

**After**

![Apr-13-2023 15-27-09](https://user-images.githubusercontent.com/8698078/231886944-b88be313-a14c-4879-9c28-14cbdb6f2607.gif)